specification/columns/chargeclass.md

```diff
@@ -1,6 +1,6 @@
 # Charge Class
 
-A Charge Class indicates whether the row represents a regular charge or a correction to one or more previous charges. Charge Class is commonly used to differentiate refunds from regularly incurred charges.
+A Charge Class indicates whether the row represents a regular charge unit based usage charge, a derivative percentage charged based on usage of one or more otehr services or a correction to one or more previous charges. Charge Class is commonly used to differentiate refunds from regularly incurred charges.

```

Comment:
I'm not quite following. Is this trying to differentiate an independent quantity vs. a percentage-based quantity? If it's percentage-based, would that include taxes? If it's percentage-based, how can we see what the percentage is? I think I like this conceptually but haven't fully grokked it.

FYI - Typo in "otehr services".

Reply:
I agree that we need to think through this more and provide more precise descriptions for each value and column description.

Taxes would fall under this additional ChargeClass category. Based on the discussions at TF-1/TF-2, in the case of tax-related charges, SkuId, SkuPriceId, PricingQuantity, etc., MUST NOT be provided. The reasoning behind this decision is that specifying what's expected in those columns for tax-related charges would introduce more confusion than value. Additionally, some providers are not providing those charge records at all. 
So, regarding taxes, this would simply indicate that it's not a standalone charge but a percentage of the cost of one or multiple Regular (or Direct) charges, which may not necessarily be of the same SKU.

So, for taxes, this would simply provide the indication that it's not a standalone charge but a percentage of the cost of one ore multiple Regular (or Direct) charges. 

A column providing information about the actual percentage (%) is still on our TODO list, and I assume it's part of the 1.x scope.

As Riley pointed out, there are Percentage charges based on one or multiple charges related to the same SkuId and SkuPriceId. However, I'm not sure what should be provided in PricingQuantity and UsageQuantity. This is another use case that we need to think through and discuss further...